infra/modules/aws-host/main.tf

```diff
@@ -0,0 +1,194 @@
+# NOTE: region used to be passed in as a variable; put it MUST match the region in which the lambda
+# is provisioned, and that's implicit in the provider - so we should just infer from the provider
```

Comment:
ok, then why passing it in variables? totally eliminate then?

Reply:
mmm, yeah; this is copy-paste from places where eliminating it would be a breaking change, but it can be removed from `aws-host` as this is a new module that no existing deployments should reference.